### ISSUE TITLE
httpdns失效

### ISSUE THREAD

**Issue by cnhappier** [aliyun/aliyun-oss-android-sdk#194]
访问 httpdns时，会有异常。Cleartext HTTP traffic to 203.107.1.1 not permitted。

HttpdnsMini，无法获取到对应ip，因此httpdns现在是无效的。

**Comment by pushuhengyang** [comment]
> Cleartext HTTP traffic to 203.107.1.1 not permitted

是的 可以参考 https://blog.csdn.net/gengkui9897/article/details/82863966
